File: spring-boot-2x/ff4j-sample-springboot2/src/main/java/org/ff4j/springboot2/config/CORSConfig.java
package org.ff4j.springboot2.config;

import org.springframework.context.annotation.Configuration;
import org.springframework.web.servlet.config.annotation.CorsRegistry;
import org.springframework.web.servlet.config.annotation.WebMvcConfigurer;

/**
 * CORS configuration
 *
 * @author <a href="mailto:drizztguen77@gmail.com">Curtis White</a>
 */
@Configuration
public class CORSConfig implements WebMvcConfigurer {

    @Override
    public void addCorsMappings(CorsRegistry registry) {
        registry.addMapping("/**");
    }
}

File: ff4j-spring-boot-samples/ff4j-spring-boot-starter-webmvc-sample/src/main/kotlin/org/ff4j/spring/boot/starter/webmvc/sample/config/CORSConfig.kt
package org.ff4j.spring.boot.starter.webmvc.sample.config

import org.springframework.context.annotation.Configuration
import org.springframework.web.servlet.config.annotation.CorsRegistry
import org.springframework.web.servlet.config.annotation.WebMvcConfigurer

@Configuration
class CORSConfig : WebMvcConfigurer {
    override fun addCorsMappings(registry: CorsRegistry) {
        registry.addMapping("/**")
    }
}
